ipld/kamt/benches/kamt_benchmark.rs

```diff
@@ -0,0 +1,120 @@
+// Copyright 2019-2022 ChainSafe Systems
```

Comment:
????

Reply:
Copy of https://github.com/filecoin-project/ref-fvm/blob/master/ipld/hamt/benches/hamt_benchmark.rs
Similar one in https://github.com/filecoin-project/ref-fvm/blob/master/ipld/amt/benches/amt_benchmark.rs